src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -133,6 +150,18 @@ export default {
 				{ id: 0 }, { id: 0 }, { id: 0 },
 				{ id: 0 }, { id: 0 }, { id: 0 },
 			],
+			fiveDollarsRowLoans: [
+				{ id: 0 }, { id: 0 }, { id: 0 },
```

Comment:
It will be easier to maintain if we auto populate an array here. Also, why 30 items?

```
new Array(30).fill({ id: 0 });
```

Reply:
Nice suggestion, thanks!
Items number is coming from prototype, https://www.figma.com/file/6RV1BiJR8s8rqAVlv6dRwl/branch/tGKlvXECOa0p9p52HjtdJb/Lending-Home-V4---Featured-Loan?type=design&node-id=3302-245&mode=design&t=OxJGescUrKR73QeO-0